projects/1.20/assets/food-expansion-reimagined/foodexpansion/lang/zh_cn.json

```
@@ -0,0 +1,44 @@
+{
+    "item.foodexpansion.jelly": "翡翠果冻",
+    "item.foodexpansion.bacon": "培根",
+    "item.foodexpansion.cooked_bacon": "香脆培根",
+    "item.foodexpansion.fried_egg": "煎蛋",
+    "item.foodexpansion.bacon_and_egg": "培根煎蛋",
+    "item.foodexpansion.carrot_seed_soup": "胡萝卜汤",
+    "item.foodexpansion.squid": "生鱿鱼",
+    "item.foodexpansion.cooked_squid": "烤鱿鱼",
+    "item.foodexpansion.compressed_flesh": "腊肉",
+    "item.foodexpansion.chocolate_bar": "巧克力棒",
+    "item.foodexpansion.spider_soup": "蛛眼炖汤",
+    "item.foodexpansion.nether_wart_soup": "下界疣炖菜",
```

```suggestion
    "item.foodexpansion.nether_wart_soup": "下界疣煲",
```
`Stew`应对应`煲`，见[原版译名标准化](https://minecraft.fandom.com/zh/wiki/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96)
此问题请全文通查